ext/afform/admin/Civi/AfformAdmin/AfformAdminMeta.php

```diff
@@ -88,7 +88,7 @@ public static function getFields($entityName, $params = []) {
       'checkPermissions' => FALSE,
       'loadOptions' => ['id', 'label'],
       'action' => 'create',
-      'select' => ['name', 'label', 'input_type', 'input_attrs', 'required', 'options', 'help_pre', 'help_post', 'serialize', 'data_type', 'entity', 'fk_entity', 'readonly', 'operators'],
+      'select' => ['name', 'label', 'input_type', 'input_attrs', 'required', 'options', 'help_pre', 'help_post', 'serialize', 'data_type', 'entity', 'fk_entity', 'readonly', 'operators', 'class_name'],
```

Comment:
`class_name` is not part of the Api4 field spec, so this addition is not needed (it won't do anything as there's nothing to retrieve)